tools/dotcms-cli/action/.github/workflows/main.yml

```
@@ -30,7 +30,7 @@ jobs:
 
   sync-with-dotcms:
     needs: dotcli-runner
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Why not use variable with fallback?

```suggestion
    runs-on: ubuntu-${{ vars.UBUNTU_RUNNER_VERSION || '22.04' }}
```

---

It isn't part of our CICD process. It's an example of using dotCLI along with Github Actions.